src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -137,238 +139,291 @@ public void AndroidStudioConverter_ConvertToSarifIssue_HintsAreStapledToDescript
             };
 
             var uut = new AndroidStudioProblem(builder);
-            Result result = AndroidStudioConverter.ConvertProblemToSarifIssue(uut);
+            Result result = new AndroidStudioConverter().ConvertProblemToSarifResult(uut);
```

Comment:
As discussed: ok as is (but `static` pattern would be good if other code needed to use it as a helper).